server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertionsAPITests.java

```diff
@@ -86,13 +85,13 @@ public class GenerateAssertionsAPITests extends TestClassWithAuth {
   private final GenerateAssertions endpoint = new GenerateAssertions();
 
   /**
-   * Mock response for tinyExample1 contest
+   * Mock response for tinyExample1 contest, which succeeded and is not worth retrying.
    */
   private final static GenerateAssertionsResponse tinyIRVResponse
       = new GenerateAssertionsResponse(tinyIRV, true, false);
 
```

Comment:
Not clear what we mean by "boring" parameters!

Reply:
I was trying to convey the idea that this is not a test for funny edge cases of those parameters.

Reply:
Have changed the wording slightly. In general it's a good question to think about how to succinctly convey "in this test the parameter I actually care about is..."